### PR TITLE
[Docs] Note about apiserver outside of cluster

### PR DESCRIPTION
**Pull request by nebril**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6995)

### CODE REVIEW

**Comment by nebril** [comment]
test-docs-please

**Comment by nebril** [comment]
test-me-please

**Review by vadorovsky — APPROVED** [review]

File: Documentation/kubernetes/troubleshooting.rst [hunk 1]
```diff
@@ -120,3 +120,12 @@ Now if you try to create a CNP as a TPR, you will get an error:
 .. code:: bash
 
        $ Error from server (BadRequest): error when creating "cilium-tpr.yaml": the API version in the data (cilium.io/v1) does not match the expected API version (cilium.io/v2)
+
+Apiserver outside of cluster
+==============================
+
+If you are running Kubernetes Apiserver outside of your cluster for some reason (like keeping master node behind firewall), make sure that you run Cilium on master node too.
```
Comment (ianvernon) [inline Documentation/kubernetes/troubleshooting.rst]: Grammar: like keeping master nodes behind a firewall

Comment (ianvernon) [inline Documentation/kubernetes/troubleshooting.rst]: make sure that you run Cilium on master nodes too.

File: Documentation/kubernetes/troubleshooting.rst [hunk 2]
```diff
@@ -120,3 +120,12 @@ Now if you try to create a CNP as a TPR, you will get an error:
...
+Otherwise kubernetes pod proxies created by apiserver will not be able to route to cluster IPs and you may encounter errors when trying to proxy traffic to pods.
```
Comment (ianvernon) [inline Documentation/kubernetes/troubleshooting.rst]: kubernetes --> Kubernetes

Comment (ianvernon) [inline Documentation/kubernetes/troubleshooting.rst]: Be consistent in casing --> use apiserver or Apiserver throughout this change

Comment (ianvernon) [inline Documentation/kubernetes/troubleshooting.rst]: are cluster IPs here the same thing as ClusterIPs? If so change the spacing / casing to match the nomenclature set forth by Kubernetes

File: Documentation/kubernetes/troubleshooting.rst [hunk 3]
```diff
@@ -120,3 +120,12 @@ Now if you try to create a CNP as a TPR, you will get an error:
...
+
```
Comment (ianvernon) [inline Documentation/kubernetes/troubleshooting.rst:129]: for the cilium DaemonSet to ensure that...

File: Documentation/kubernetes/troubleshooting.rst [hunk 4]
```diff
@@ -120,3 +120,12 @@ Now if you try to create a CNP as a TPR, you will get an error:
...
+You may run Cilium as a `static pod <https://kubernetes.io/docs/tasks/administer-cluster/static-pod/>`_ or set `tolerations <https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/>`_ for Cilium daemonset to make sure,
```
Comment (ianvernon) [inline Documentation/kubernetes/troubleshooting.rst]: Cilium pods will be scheduled on your master nodes.

**Review by ianvernon — DISMISSED** [review]
Good idea to add this change! Some misc. grammatical / casing nitpicks.

**Comment by nebril** [comment]
test-docs-please

**Comment by nebril** [comment]
test-me-please

**Pull request merged**